tests/product/tests/test_process_api_server_rules.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
Is there a reason to create a test file for each process, from first sight logic looks quite similar?